config/routes.rb

```diff
@@ -361,7 +361,8 @@
       get '/persons/:wca_id/results' => "persons#results", as: :person_results
       get '/persons/:wca_id/competitions' => "persons#competitions", as: :person_competitions
       get '/persons/:wca_id/personal_records' => "persons#personal_records", as: :personal_records
-      get '/geocoding/search' => 'geocoding#get_location_from_query', as: :geocoding_search
+      get '/geocoding/search' => 'geocoding#location_from_query', as: :geocoding_search
```

Comment:
Do we really want to keep these routes in the public API? Maybe a post request with a csrf check would be better

Reply:
The geocoding routes are restricted via `before_action :raise_if_invalid`.
Otherwise, someone would have found out that it's possible to piggyback our Google Maps geocoder for free years ago